stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -102,9 +135,9 @@ class Worksheet(_WorkbookChild):
         values_only: bool,
     ) -> Generator[tuple[Cell | str | float | datetime | None, ...], None, None]: ...
     @property
-    def rows(self) -> Generator[Cell, None, None]: ...
+    def rows(self) -> Generator[tuple[Cell, ...], None, None]: ...
     @property
-    def values(self) -> Generator[str | float | datetime | None, None, None]: ...
+    def values(self) -> Generator[tuple[str | float | datetime | None, ...], None, None]: ...
```

Comment:
@Avasam's annotations for these are slightly different:

https://github.com/python/typeshed/pull/9511/files#diff-dabc24e406450b802c51dd6c9d1062ac0fed975dc4daef0e9cbe468aedcb6978L103-R114